prow/scripts/cluster-integration/kyma-gardener-integration.sh

```diff
@@ -160,11 +160,6 @@ kyma version
 
 shout "Running Kyma tests"
```

Comment:
I'm ok with current fix. But the final scenario should be that this custom code is removed and the [kyma-testing.sh](https://github.com/kyma-project/test-infra/blob/master/prow/scripts/kyma-testing.sh) is used. 


If the kyma-testing.sh was used from the beginning then this error would not occur at all